buildSrc/deploy.gradle

```diff
@@ -60,47 +69,94 @@ ext.configureDeploy = { context ->
-                    def artifactVersion = Components.getModuleVersion(project.name)
-                    println "uploadAcrhives $artifactName $artifactVersion"
-
-                    pom.groupId = artifactGroupId
-                    pom.artifactId = artifactName
-                    pom.version = artifactVersion
-
-                    pom.project {
-                        dependencies {
-                            Components.getAndroidStandardDependencies(project.name).each {
-                                def depArtifactName = it.getArtifactName()
-                                def depArtifactVersion = Components.getModuleVersion(it.getName())
-                                dependency {
-                                    groupId artifactGroupId
-                                    artifactId depArtifactName
-                                    version depArtifactVersion
-                                    scope "compile"
-                                }
+                    // If -PdeployOnlyIfNotExist=true deploy only artifacts that don't exist in artifactory
+                    if (project.hasProperty(GradleProperties.DEPLOY_ONLY_IF_NOT_EXIST)) {
+                        if (project.property(GradleProperties.DEPLOY_ONLY_IF_NOT_EXIST) == trueValue
+                                && Artifactory.isLibraryAlreadyDeployed(project.name)
+                        ) return
+                    }
+
+                    groupId = artifactGroupId
+                    artifactId = artifactName
+                    version = artifactVersion
+
+                    from components.release
+
+                    pom {
+                        name = artifactName
+                        description = artifactDescription
+                        url = artifactUrl
+
+                        licenses {
+                            license {
+                                name = licenseName
+                                url = licenseUrl
                             }
                         }
-                    }
-                    //This code deletes "implementation project("...") dependencies from pom file"
-                    pom.whenConfigured {
-                        p ->
-                            p.dependencies = p.dependencies.findAll {
-                                dep -> dep.groupId != rootDir.name
+
+                        scm {
+                            connection = scmConnection
+                            developerConnection = scmDeveloperConnection
+                            url = scmUrl
+                        }
+
+                        developers {
+                            developer {
+                                name = developerName
+                                email = developerMail
+                                organization = developerOrganization
+                                organizationUrl = developerOrganizationUrl
```

Comment:
https://medium.com/@nmauti/sign-and-publish-on-maven-central-a-project-with-the-new-maven-publish-gradle-plugin-22a72a4bfd4b organization, issueManagement